hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -140,6 +143,63 @@ private OmBucketInfo(String volumeName,
     this.quotaInNamespace = quotaInNamespace;
   }
 
+  /**
+   * Private constructor, constructed via builder.
+   * @param volumeName - Volume name.
+   * @param bucketName - Bucket name.
+   * @param acls - list of ACLs.
+   * @param isVersionEnabled - Bucket version flag.
+   * @param storageType - Storage type to be used.
+   * @param creationTime - Bucket creation time.
+   * @param modificationTime - Bucket modification time.
+   * @param metadata - metadata.
+   * @param bekInfo - bucket encryption key info.
+   * @param sourceVolume - source volume for bucket links, null otherwise
+   * @param sourceBucket - source bucket for bucket links, null otherwise
+   * @param usedBytes - Bucket Quota Usage in bytes.
+   * @param quotaInBytes Bucket quota in bytes.
+   * @param quotaInNamespace Bucket quota in counts.
+   * @param ecReplicationConfig EC replication config.
+   */
+  @SuppressWarnings("checkstyle:ParameterNumber")
+  private OmBucketInfo(String volumeName,
```

Comment:
Same as above, I don't think we need to duplicate this constructor to add an extra parameter, as its private and is only used by the builder.